### PR TITLE
Fix layout issues with install button

### LINKED ISSUE #9669: There's no spacing between the install button and the install warning message when the Download file link is missing
Follow up for #4910 

### Describe the problem and steps to reproduce it:
1. Open the following extension detail page - https://addons-dev.allizom.org/en-US/firefox/addon/enhancer-for-youtube-2/
2. Notice the add-on detail card layout 

### What happened?
There's no spacing between the install button and the install warning message

### What did you expect to happen?
There should be some space between the install button and the install warning message

### Anything else we should know?
Note that this issue reproduces only when the summary extends to just one line, like in the example provided

image

image

### LINKED ISSUE #9670: Add more space between the Download file link and the install warning message
Follow up for #4910 

### Describe the problem and steps to reproduce it:
1. Open the following extension detail page - https://addons-dev.allizom.org/en-US/firefox/addon/signature-files/
2. Notice the add-on detail card layout

### What happened?
There's too little space between the Download file link and the installation warning message

### What did you expect to happen?
Some extra space between the Download file link and the installation warning message should be added

### Anything else we should know?
This issue does not apply to the mobile view

image

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #9670 
Fixes #9669 

Screen Shot 2020-09-21 at 2 19 58 PM
Screen Shot 2020-09-21 at 2 17 55 PM
Screen Shot 2020-09-21 at 2 17 02 PM

### CODE REVIEW

File: src/amo/components/InstallButtonWrapper/index.js [hunk 1]
```diff
@@ -118,7 +125,7 @@ export const InstallButtonWrapperBase = (props: InternalProps) => {
           buttonType={getFirefoxButtonType}
           className={className ? `GetFirefoxButton--${className}` : ''}
         />
-        {!isCompatible && installURL ? (
+        {showDownloadLink ? (
           <div className="InstallButtonWrapper-downloadLink">
             <a
               className="InstallButtonWrapper-downloadLink-link"
```
Comment (muffinresearch) [inline src/amo/components/InstallButtonWrapper/index.js:131]: This is a nit-pick (especially as it wasn't added here) but the `Link` in `downloadLink` feels superfluous.

File: src/amo/components/InstallButtonWrapper/styles.scss [hunk 2]
```diff
@@ -9,7 +9,7 @@
 }
 
 .InstallButtonWrapper-downloadLink {
-  margin-top: 12px;
+  margin: 12px auto;
```
Comment (muffinresearch) [inline src/amo/components/InstallButtonWrapper/styles.scss:12]: Any reason to use `auto` rather than `0`?

**Review by muffinresearch — APPROVED** [review]
r+wc

File: src/amo/components/InstallButtonWrapper/index.js (hunk 1 above)
Comment (bobsilverberg) [reply src/amo/components/InstallButtonWrapper/index.js:131]: Yeah, I think we can change the parent to just `InstallButtonWrapper-download`.

File: src/amo/components/InstallButtonWrapper/styles.scss (hunk 2 above)
Comment (bobsilverberg) [reply src/amo/components/InstallButtonWrapper/styles.scss:12]: Nope!

**Pull request merged**